### PR TITLE
Added showlist function, inventory, dialog, items, places

### PR DESCRIPTION
**Pull request by prachi24s**

### CODE REVIEW

**Review by swathipriya123 — APPROVED** [review]
Need to add position for james

File: start_camelot.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+    elif i == 'input Exit_Tavern Tavern.Exit':
+         exit_place('Kate', 'Tavern.BackDoor')
+         enter_place('Kate', 'Courtyard.Gate')
+     
+    elif i == 'input Talk_Jerry Jerry':
+         talktojerry('Kate', 'Jerry')
+         
+         
+    elif i == 'input Talk_James James':
```
Comment (ckarthala) [inline start_camelot.py:76]: here we have switch statement code smell. we need to fix it

**Review by ckarthala — COMMENTED** [review]
here we have switch statement code smell. we need to fix it

**Review by ckarthala — APPROVED** [review]

File: camelot_utiles.py [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    character_clothing('Mike', 'Long', 'HeavyArmour')
+    enableIcon('Talk_Mike', 'Talk', 'Mike', "Talk to the mike", True)
+    enableIcon('Exit_CastleCrossroads', 'Open', 'CastleCrossroads.Gate', 'Enter the CastleBedroom', True)
+    
+    create_place('CastleBedroom', 'CastleBedroom')
+    create_character('James', 'B')
+    character_clothing('James', 'Mage_Beard', 'King')
+    set_character_position('James', 'CastleBedroom.Bed')
+    enableIcon('Talk_James', 'Talk', 'James', "Talk to the James", True)
+    create_item('GoldCup', 'GoldCup')
```
Comment (vasanthhny) [inline camelot_utiles.py:117]: The code looks clean and good, can you add visual effect for the king at castle bedroom when he give the goldcup to the player.

**Review by vasanthhny — APPROVED** [review]

**Pull request merged**